tensorboard/plugins/mesh/README.md

```diff
@@ -92,6 +92,20 @@ camera_config = {
 Keep in mind that scene configuration is not a trainable variable (i.e., it is 
 a static attribute) and is fixed at summary-creation time.
 
+## Example
+
+Please refer to the demo application `demo_utils.py` as an example of how to
+use the plugin and display a mesh with colors in TensorBoard. Note, that demo
+application will read PLY file in ASCII format and doesn't support all
+the variations of PLY format. You can find a sample mesh in PLY format [stored
```

Comment:
Users running the demo already have the test data file and don’t need to
fetch anything from the cloud. Can we point them to the `test_data/...`
file instead of or in addition to this link?